.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -8,13 +8,14 @@ assignees: ''
 ---
 
 **Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+Describe what the problem is. For example, “I'm always frustrated when [...]”.
```

Comment:
Maybe all these could be in block comments so that these descriptions doesn't comes up in the .md format but it comes up when someone edits it?

Reply:
Sure. I can do that.

Reply:
Added in new commit.